parsing/pprintast.ml

```diff
@@ -268,16 +273,21 @@ let iter_loc f ctxt {txt; loc = _} = f ctxt txt
 
 let constant_string f s = pp f "%S" s
 
-let tyvar ppf s =
+let tyvar_name s =
```

Comment:
I find the new function name `tyvar_name` confusing. If I was asked to guess what is the "name" of a tyvar, I would say that `foo` is the name of the tyvar `'foo`. This function is in fact doing the opposite thing.